01-node-tutorial/answers/03-modules.js

```diff
@@ -0,0 +1,14 @@
+const Melanie = 'Melanie'
+
+const sayHi = (name) => {
```

Comment:
instead of this being a function inside this file it should be imported from 05-utils like: const sayHi = require('.05-utils')
it's just to teach importing modules, but you did a good job writing the function!